validator/src/main/java/org/icann/rdapconformance/validator/workflow/profile/rdap_response/general/ResponseValidationHelp_2024.java

```
@@ -0,0 +1,105 @@
+    public ResponseValidationHelp_2024(RDAPValidatorConfiguration config, RDAPValidatorResults results) {
+        super(results);
+        this.config = config;
+    }
+
+    @Override
+    public String getGroupName() {
+        return "rdapResponseHelp_2024_Validation";
+    }
+
+    @Override
+    protected boolean doValidate() throws Exception {
+        boolean isValid = true;
+        HttpResponse<String> httpHelpResponse = null;
+
+        logger.debug("Creating help query for host {}", config.getUri().getHost());
+        try {
+            var queryType = RDAPHttpQueryTypeProcessor.RDAPHttpQueryType.getType(this.config.getUri().toString());
+            String helpUri = null;
+            if (Objects.nonNull(queryType)) {
+                helpUri = CommonUtils.replaceQueryTypeInStringWith(queryType, this.config.getUri().toString(), "help");
+                int index = helpUri.indexOf("help");
+                if (index != -1) {
+                    helpUri = helpUri.substring(0, index + "help".length());
+                } else {
+                    logger.debug("Help word was not found, using original url");
+                    helpUri = this.config.getUri().getHost().concat(HELP);
+                }
+                logger.debug("Help URI built {}", helpUri);
+            } else {
+                helpUri = this.config.getUri().getHost().concat(HELP);
+            }
+
+            logger.info("Making request to: {}", helpUri);
+            HttpResponse<String> response = null;
+
+            response = RDAPHttpRequest.makeHttpGetRequest(new URI(helpUri), this.config.getTimeout());
+            int status = response.statusCode();
+            StatusCodes.add(status);
+
+            // final response
+            httpHelpResponse = response;
+            return validateHelpQuery(httpHelpResponse, isValid);
+        } catch (Exception e) {
+            logger.debug("Help query was not able to be called"); // catch for all subclasses of these exceptionn
```

this part: `// catch for all subclasses of these exceptionn`
isn't really a thing - you are catching everything here and I don't think you should ... we need to let the exception bubble up.